### DISCUSSION TITLE
Is tgtg still working for you?

### DISCUSSION THREAD

**Discussion by MrTeal** [Chouffy/home_assistant_tgtg#88] [Q&A] [answered]
Just wondering, after the latest update of home assistant (2023.10.5) tgtg is no longer starting for me. Log says:
homeassistant.exceptions.HomeAssistantError: The system cannot restart because the configuration is not valid: Platform error sensor.tgtg - Requirements for tgtg not found: ['tgtg==0.16.0'].

Anybody else having this problem?
It was working on 2023.10.4.

**Comment by Chouffy** [comment, marked as answer]
It works just fine for me in 2023.10.5
Given your error, did you try to reboot (not restart) or power off/power on your Home Assistant?
It makes complain a bit, but try it anyway

**Reply by MrTeal** [reply]
Thanks for your reply! I did a reboot and now my HA won't start, seems to be having this issue
https://github.com/home-assistant/core/issues/73885

So probably something else is broken.. 
Anyway, thank you for your help.
